c_glib/arrow-glib/table.h

```diff
@@ -89,5 +89,8 @@ GArrowTable    *garrow_table_replace_column(GArrowTable *table,
                                             GError **error);
 gchar          *garrow_table_to_string     (GArrowTable *table,
                                             GError **error);
+GArrowTable *garrow_table_concatenate(GArrowTable *table,
```

Comment:
Could you add `GARROW_AVAILABLE_IN_0_14`?

Reply:
Yes. I've added it.